### PR TITLE
Update winning-over-clients.md

### PR DESCRIPTION
**Pull request by monizb**
Addition of points

## Description

Added 3 more advantages to the list which points towards the speed, scalability and the plugin architecture of Gatsby.

### Documentation

Original Documnetation at:
https://www.gatsbyjs.org/docs/winning-over-stakeholders/

@gatsbyjs/learning
@laurieontech

### CODE REVIEW

File: docs/docs/winning-over-clients.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ Some benefits that you can use to pitch Gatsby to clients include:
 - Fast development speed ensures project will be on time
 - Ease of making development changes helps keep project under budget
 - Lower hosting cost helps keep project under budget
+- Fast performing websites which reduces page load and response time
```
Comment (laurieontech) [inline docs/docs/winning-over-clients.md]:
```suggestion
- Fast performing websites which reduce page load and response time
```

File: docs/docs/winning-over-clients.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ Some benefits that you can use to pitch Gatsby to clients include:
...
+- Helps in making highly scalable and dynamic apps
+- Rich plugin ecosystem which helps tackle complex codes
```
Comment (laurieontech) [inline docs/docs/winning-over-clients.md]:
```suggestion
- Rich plugin ecosystem which helps tackle complex code
```

File: docs/docs/winning-over-clients.md [hunk 3]
```diff
@@ -13,6 +13,9 @@ Some benefits that you can use to pitch Gatsby to clients include:
 - Fast development speed ensures project will be on time
 - Ease of making development changes helps keep project under budget
 - Lower hosting cost helps keep project under budget
+- Fast performing websites which reduce page load and response time
+- Helps in making highly scalable and dynamic apps
```
Comment (laurieontech) [inline docs/docs/winning-over-clients.md]:
```suggestion
- Supports creation of highly scalable and dynamic apps
```

**Comment by monizb** [comment]
> Thanks for the PR and the additional benefits! Made a couple of edits.

Hey!
Thank you for the changes!

**Pull request merged**